src/AbstractOptions.php

```diff
@@ -33,7 +38,7 @@ abstract class AbstractOptions implements ParameterObjectInterface
     /**
      * Constructor
      *
-     * @param  array|Traversable|null $options
+     * @param  iterable<string, TValue>|AbstractOptions<TValue>|null $options
```

Comment:
I wouldn't really expand this signature, unless we are testing for it ourselves

Reply:
I added it because of an existing test - you'll see it calls fromArray or something, that already has this signature

Reply:
Thanks for checking :+1: